### DISCUSSION TITLE
New timelock feature

### DISCUSSION THREAD

**Discussion by aesedepece** [witnet/sheikah#1701] [General]
This is a discussion thread for providing UX feedback for the new timelock feature, which was available on the wallet backend but not on Sheikah's UI.

**Comment by aesedepece** [comment]
My two cents:

- IMO, timelocks are an advanced option — most times you don't want to put a timelock on your transaction. For the unsophisticated user, seeing this in the VTT sending modal can be confusing. I'd hide it inside an "Advanced options" thingy.
- The date picker for the timelock is localized to Spanish even if I have my Sheikah and my OS set to English.

**Reply by gabaldon** [reply]
https://github.com/witnet/sheikah/pull/1702

**Comment by aesedepece** [comment]
When setting a timelock, it is unclear to me whether the input is UTC or local. From a user perspective, local is ideal and UTC is also OK. Regardless of what we use, the important thing is that we clearly state it in the UI.
